aws/aws-how-to/instances/launch-ubuntu-desktop.rst

```diff
@@ -1,78 +1,105 @@
-Launch an Ubuntu desktop on EC2
-===============================
+Launch and connect to an Ubuntu Mantic desktop EC2 instance
```

Comment:
Let's change this to 'Launch an Ubuntu Mantic desktop on EC2'. We want to try and minimise the number of words used, while still conveying things accurately. We want the title to include important words to help with SEO, but at the same time we don't want it to be too long or unwieldy to read. 

Such decisions are quite subjective, but that's okay since the whole documentation process is quite iterative and we'll keep improving things in the future.